### ISSUE TITLE
Upstream type-scoped context impact

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#231]
This issue https://github.com/w3c/json-ld-syntax/issues/174 for JSON-LD 1.1 doesn't directly affect /our/ context, but we should set it up such that others aren't affected by it when importing linked art into their own contexts.

The issue can be deferred as it doesn't directly affect the model or representation, but needs to be taken into account once JSON-LD 1.1 lands.

**Comment by azaroth42** [comment]
Closing, as the upstream issue is resolved.
